contracts/treasury/priceOracle/KyberDmmChainLinkPriceOracle.sol

```diff
@@ -0,0 +1,477 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity 0.7.6;
+
+import {ILiquidationPriceOracleBase} from '../../interfaces/liquidation/ILiquidationPriceOracleBase.sol';
+import {IChainLinkAggregatorProxy} from '../../interfaces/liquidation/thirdParty/IChainLinkAggregatorProxy.sol';
+import {IDMMPool} from '../../interfaces/liquidation/thirdParty/IDMMPool.sol';
+import {PermissionAdmin, PermissionOperators} from '@kyber.network/utils-sc/contracts/PermissionOperators.sol';
+import {Utils} from '@kyber.network/utils-sc/contracts/Utils.sol';
+import {IERC20Ext} from '@kyber.network/utils-sc/contracts/IERC20Ext.sol';
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+import {EnumerableSet} from '@openzeppelin/contracts/utils/EnumerableSet.sol';
+
+
+/**
+* @dev Contract to calculate expected return amounts for a liquidation call
+*   Also work with Kyber Dmm LP tokens
+*   Can use hint to:
+*     1. Remove liquidity given LP tokens
+*     2. Calculate price of LP tokens to a dest token
+*     3. Calculate price of normal tokens to a dest token
+*   It may not work for LPs of token with fees
+*/
+contract KyberDmmChainLinkPriceOracle is ILiquidationPriceOracleBase, PermissionAdmin, PermissionOperators, Utils {
```

Comment:
it 's redundant to inherit `PermissionAdmin`

Reply:
Removed